plotnine/aes.py

```diff
@@ -371,14 +372,24 @@ def is_position_aes(vars_):
         return aes_to_scale(vars_) in {'x', 'y'}
 
 
+def _make_label(ae, label):
+    if isinstance(label, pd.Series):
+        return label.name
+    # if label is a scalar
+    elif not isinstance(label, Iterable) or isinstance(label, str):
+        return strip_calculated_markers(str(label))
+    else:
+        return None
```

Comment:
Make `make_label` an internal function in `make_labels`.

Reply:
done!